.github/workflows/ansible-release.yml

```diff
@@ -0,0 +1,88 @@
+name: ansible-release
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: >-
+          Release Version. Example : 11.1.0
+        required: true
+      ansible-major-version:
+        description: Example 11
+        required: true
+env:
+  CI_COMMIT_MESSAGE: 'Ansible ${{ inputs.ansible-version }}: Dependencies, changelog and porting guide'
+
+
+jobs:
+  build:
```

Comment:
What's the difference between the build job in this workflow and the other one? Why do we need two workflows here?

Reply:
There are two steps right now (in the current process).
- This job creates the .deps, .tags and .yaml files required for the release.
- After the (manual) merging of the PR created by the aforesaid steps we run the second job to build the source tarball and wheel, upload it to PyPI and create the git tag.

Reply:
@anweshadas does that sdist need to contain the merged PR contents or can it be built ahead of time?

Reply:
> @anweshadas does that sdist need to contain the merged PR contents or can it be built ahead of time?

The `sdist` should not be build prior to the merging of the PR as we verify the `deps` file during the PR review.

Reply:
Why?

Reply:
I mean, what's missing? Can't it be built anyway and discarded if the review result wasn't acceptable?

Reply:
The current version of this job is uploading both sdist and wheel.

Reply:
@felixfontein I think my initial concern was regarding building the artifacts (both), not uploading (which is to be a separate thing).